dashboard/app/views/plc/user_course_enrollments/index.html.haml

```diff
@@ -4,37 +4,31 @@
 - if @user_course_enrollments.nil? || @user_course_enrollments.empty?
   %p Hello #{current_user.name}, your future course progress will go here
 - else
-  %p Hello #{current_user.name}, here is your progress for your courses
-
   - @user_course_enrollments.each do |user_course_enrollment|
-    %h2 #{user_course_enrollment.plc_course.name}
-    - user_course_enrollment.plc_unit_assignments.each do |enrollment_unit_assignment|
-      - next if enrollment_unit_assignment.status == Plc::EnrollmentUnitAssignment::START_BLOCKED
-
-      - content_module_assignment = enrollment_unit_assignment.module_assignment_for_type Plc::LearningModule::CONTENT_MODULE
-      - practice_module_assignment = enrollment_unit_assignment.module_assignment_for_type Plc::LearningModule::PRACTICE_MODULE
-
-      .outlined_section
-        %h4
-          = enrollment_unit_assignment.plc_course_unit.unit_name
-        .course_unit_sections
-          - evaluation_complete = content_module_assignment.nil?
-          .course_unit_section{class: evaluation_complete ? 'completed' : 'not_started'}
-            %i.fa.fa-info-circle
-            .course_unit_section_name
-              - if evaluation_complete
-                = link_to 'Unit Overview', plc_enrollment_unit_assignment_path(enrollment_unit_assignment), class: 'course_unit_link'
-              - else
-                = link_to 'Unit Overview', perform_evaluation_path(enrollment_unit_assignment), class: 'course_unit_link'
-          .course_unit_section{class: content_module_assignment.try(:status) || 'not_started'}
-            %i.fa.fa-book
-            .course_unit_section_name
-              = link_to 'Unit Content', plc_enrollment_unit_assignment_path(enrollment_unit_assignment),
-                class: 'course_unit_link'
-          .course_unit_section{class: practice_module_assignment.try(:status) || 'not_started'}
-            %i.fa.fa-graduation-cap
-            .course_unit_section_name
-              = link_to 'Teaching Practices', plc_enrollment_unit_assignment_path(enrollment_unit_assignment),
-                class: 'course_unit_link'
-      %br
-    %br
+    %h3.course_title
+      = user_course_enrollment.plc_course.name
+    -# Lame that we have to do this but Firefox doesn't support multiline flexboxes
+    - user_course_enrollment.plc_unit_assignments.each_slice(2).to_a.each do |unit_group|
+      .course_unit_sections
+        - unit_group.each do |unit_enrollment|
+          .course_unit_section
+            .course_unit_title
+              = unit_enrollment.plc_course_unit.unit_name
+            - if unit_enrollment.status == Plc::EnrollmentUnitAssignment::START_BLOCKED
+              .coming_soon
+                Coming soon!
+            - else
+              - required_module_assignment = unit_enrollment.module_assignment_for_type Plc::LearningModule::REQUIRED_MODULE
```

Comment:
I thought about factoring it into iterators but it doesn't really save much. Content and practice modules are similar but Unit Overview (and Peer Review when its ready) aren't the same as it.

Let me mess around with aliases